Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 ruby '2.7.1'
 
 gem 'puma'
-gem 'rails', '~> 6'
+gem 'rails', '~> 6.1.3.1'
 gem 'pg', '0.21.0' # Fix version until Rails supports 1.0.0
```

Comment:
The build is failing. Seems like there is a clash in the postgres dependency version. The rails activerecord postgresql adapter requires pg to be ~> 1.1 but our gemfile fixes pg to 0.21.0. 

Can you set pg to `'~> 1'`? I think it should be safe to upgrade.